gradle/testing.gradle

```diff
@@ -118,9 +142,16 @@ tasks.withType(Test) { theTask ->
     configureTestTask('allTest', theTask)
     configureTestTask(theTask.name, theTask)
     testLogging {
-        events 'started', 'passed', 'failed', 'skipped'
+        events 'failed'
```

Comment:
I haven't tested this with failing tests yet, so I thought it would be good to leave the existing failure log